src/components/Pool/PairsList/Pair/Pair.tsx

```diff
@@ -0,0 +1,158 @@
+  dayLiquidity?: string
+}
+
+export function Pair({
+  token0,
+  token1,
+  usdLiquidity,
+  apy,
+  containsKpiToken,
+  pairOrStakeAddress,
+  hasFarming,
+  dayLiquidity,
+  isSingleSidedStakingCampaign,
+  ...rest
+}: PairProps) {
+  const { volume24hUSD, loading } = usePair24hVolumeUSD(pairOrStakeAddress, isSingleSidedStakingCampaign)
+  const { t } = useTranslation()
+  const theme = useTheme()
+  const isMobile = useIsMobileByMedia()
+
+  const correctLogo = () =>
+    isSingleSidedStakingCampaign ? (
+      <CurrencyLogo size={isMobile ? '32px' : '45px'} marginRight={14} currency={token0} />
+    ) : (
+      <DoubleCurrencyLogo
+        spaceBetween={0}
+        marginLeft={0}
+        marginRight={14}
+        top={0}
+        currency0={token0}
+        currency1={token1}
+        size={isMobile ? 32 : 45}
+      />
+    )
+
+  return (
+    <GridCard
+      {...rest}
+      data-testid="pair-card"
+      flexWrap="wrap"
+      justifyContent="space-between"
+      padding={isMobile ? '22px 10px' : '22px'}
+    >
+      <Flex flex={isMobile ? '' : '25%'} flexDirection="row" alignItems="center">
+        {correctLogo()}
+        <Flex flexDirection="column">
+          <EllipsizedText color="white" lineHeight="20px" fontWeight="700" fontSize="16px" maxWidth="145px">
+            {unwrappedToken(token0)?.symbol}
+          </EllipsizedText>
+          <EllipsizedText color="white" lineHeight="20px" fontWeight="700" fontSize="16px" maxWidth="145px">
+            {!isSingleSidedStakingCampaign && unwrappedToken(token1)?.symbol}
+          </EllipsizedText>
+        </Flex>
+      </Flex>
+      <Flex flex={isMobile ? '' : '25%'} flexDirection="column" alignItems="flex-start" justifyContent="space-evenly">
+        <Flex style={{ gap: '6px' }} flexDirection="row" alignItems="flex-start" flexWrap="wrap">
+          <FarmingBadge isGreyed={!hasFarming}>
+            <FarmingLogo />
+            <BadgeText>{t('farming')}</BadgeText>
+          </FarmingBadge>
+          <CarrotBadge isGreyed={!containsKpiToken} />
+        </Flex>
+      </Flex>
+      <Flex flex={isMobile ? '100%' : '45%'} justifyContent="space-between">
+        <Flex alignItems="center" flex={isMobile ? '' : '30%'}>
+          <ResponsiveValueWithLabel title="TVL" value={`$${formatCurrencyAmount(usdLiquidity).split('.')[0]}`} />
+        </Flex>
+        <Flex alignItems="center" flex={isMobile ? '' : '30%'}>
+          <ResponsiveValueWithLabel
+            title="24h Volume"
+            value={`$${
+              !loading && volume24hUSD
+                ? formatCurrencyAmount(volume24hUSD).split('.')[0]
+                : dayLiquidity
+                ? dayLiquidity
+                : '-'
+            }`}
+          />
+        </Flex>
+        <Flex alignItems="center" flex={isMobile ? '' : '10%'}>
+          <ResponsiveValueWithLabel title="APY" value={`${apy.toFixed(0)}%`} fontSize="18px" color={theme.white} big />
+        </Flex>
```

Comment:
This component also this is way overused....I think the way you are styling it is actually doing it diservice since some of those components should be one styled component and we are not adhering to the dry principle..... you are repeating same styles which should be shared in one component...

Reply:
I refactored it to styled components. Fixed in e3c3d31d07e8b6cb044867257f05447473ba1d96